README.md

```diff
@@ -88,6 +88,12 @@ For more usage:
 $ docker/run_docker.sh -h
 ```
 
+## Centos 7.x
+### Binary    Download
+| Platform   | Architecture | URL                                                          |
+| ---------- | --------     | ------                                                       |
+| GNU/Linux  | 64-bit Intel | http://storage.jd.com/chubaofsrelease/chubaofs_v1.5.0.tar.gz |
```

Comment:
storage.jd.com is not the right place to put the binaries. 
see https://help.github.com/en/github/managing-large-files/distributing-large-binaries

Reply:
@wding109  the binaries file very big, is suit ?